src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1,18 +1,19 @@
 {-# LANGUAGE CPP #-}
-{-# LANGUAGE TypeApplications #-}
-{-# language DerivingStrategies, GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE ConstraintKinds #-}
 {-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
This was changed from running `stylish-haskell` - Not sure if intended? notably running stylish-haskell naiively on this file runs into the error: `src/Database/Esqueleto/Internal/Internal.hs: RealSrcSpan SrcSpanPoint "src/Database/Esqueleto/Internal/Internal.hs" 412 1: lexical error in string/character literal at character 's'`. Removing the line and running it seems to allow it to run